### PR TITLE
Pyright extended bump

### PR DESCRIPTION
**Pull request by airportyh**
Why
===

Update pyright-extended to 1.1.319.

What changed
============

Changed version and hash.

### CODE REVIEW

File: modules.json [hunk 1]
```diff
@@ -219,6 +219,10 @@
     "commit": "7266cd2c087353e75787d3f5f141eb961b71162f",
     "path": "/nix/store/1hmdmhfjv2513nk75g2aw63776gvjcjd-replit-module-python-3.10"
   },
+  "python-3.10:v13-20230712-4ba5dba": {
+    "commit": "4ba5dba393f9a1dde13b73b94a58b344ca0cc9ac",
+    "path": "/nix/store/sip8bp1wzdfyrx4j8ws9dpkczx2wf0b7-replit-module-python-3.10"
+  },
```
Comment (jackyzha0) [inline modules.json:225]: Is this change intended?

**Review by jackyzha0 — APPROVED** [review]

File: modules.json (hunk 1 above)
Comment (airportyh) [reply modules.json:225]: Yeah. I forgot to tell you: we run `python scripts/lock_module.py` after any change to a module to update the `modules.json` lock file. More details: https://github.com/replit/nixmodules#lock-modules
